app/models/survey_response.rb

```diff
@@ -63,19 +63,25 @@ def get_distance
       dest_lat, dest_lng = lat_lng(school.wgs84_lat, school.wgs84_lng)
 
       # this needs to be refactored. this is lifted from the old app.
-      url = URI("http://maps.googleapis.com/maps/api/directions/json?sensor=false&origin=#{origin_lat},#{origin_lng}&destination=#{dest_lat},#{dest_lng}")
-      http = Net::HTTP.new(url.host, url.port)
+      url = URI("https://maps.googleapis.com/maps/api/directions/json?sensor=false&origin=#{origin_lat},#{origin_lng}&destination=#{dest_lat},#{dest_lng}&key=#{Rails.application.secrets.maps_api_key}")
```

Comment:
what about
```
google_url_string = "https://maps.googleapis.com/maps/api/directions/json?" +
                    "sensor=false&" +
                    "origin=#{origin_lat},#{origin_lng}&" +
                    "destination=#{dest_lat},#{dest_lng}&" +
                    "key=#{Rails.application.secrets.maps_api_key}"
url = URI(google_url_string)
```
or something like that for readability?

Reply:
Nice :ok_hand: